docs/user/security/audit-logging.asciidoc

```diff
@@ -248,6 +332,74 @@ Refer to the corresponding {es} logs for potential write errors.
 | `success` | User has accessed an alert as part of a search operation.
 | `failure` | User is not authorized to access alerts.
 
+.2+| `case_get`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_resolve`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_find`
+| `success` | User has accessed a case as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_ids_by_alert_id_get`
```

Comment:
This one isn't technically a search operation but I wasn't sure how to describe it. Thoughts @cnasikas ?

Reply:
It looks like this is associated with https://www.elastic.co/guide/en/kibana/current/cases-api-get-cases-by-alert.html whereas `case_get` is associated with https://www.elastic.co/guide/en/kibana/current/cases-api-get-case.html. Is there a reason one of those is considered a "search operation" and the other isn't? Knowing that might help influence the word choice.

Reply:
Yeah I wasn't sure about this one. I guess it depends how we define a search haha. The `case_ids_by_alert_id_get` leverages an aggregation to retrieve the case ids, so it's not exactly a traditional "find" operation but it's more than simply a "get" like `case_get`. I think I'll remove the search wording though since we aren't actually calling `find` under the scenes.